WalletWasabi.Fluent/Views/AddWallet/Create/MultiShareOptionsView.axaml

```diff
@@ -0,0 +1,55 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:create="using:WalletWasabi.Fluent.ViewModels.AddWallet.Create"
+             mc:Ignorable="d" d:DesignWidth="428" d:DesignHeight="371"
+             x:CompileBindings="True"
+             x:DataType="create:MultiShareOptionsViewModel"
+             x:Class="WalletWasabi.Fluent.Views.AddWallet.Create.MultiShareOptionsView">
+  <ContentArea Title="{Binding Title}"
+               Caption="Set number of shares and threshold."
+               CancelContent="Cancel"
+               EnableCancel="{Binding EnableCancel}"
+               EnableBack="{Binding EnableBack}"
+               EnableNext="True" NextContent="Continue" FocusNext="True"
+               ScrollViewer.HorizontalScrollBarVisibility="Auto">
+    <StackPanel HorizontalAlignment="Center"
+                VerticalAlignment="Center">
+      <!-- Number of shares in which the seed will be split. -->
+      <Label Content="Shares:"
+             Target="SharesNumericUpDown" />
+      <DockPanel VerticalAlignment="Center">
+        <InfoMessage DockPanel.Dock="Right"
+                     VerticalAlignment="Center"
+                     Margin="14,0,0,0"
+                     ToolTip.Tip="This sets the total number of pieces (or 'shares') your secret will be split into. Each share can be given to a different trusted party." />
```

Comment:
for me the tooltip doesn't work.
neither at `threshold`

Reply:
the issues with tooltips is related to Dialogs.axaml background animation https://github.com/WalletWasabi/WalletWasabi/blob/0f649ab5b106ed45008f8b1044a643a9a5698093/WalletWasabi.Fluent/Views/Shell/Dialogs.axaml#L21 and it was there before, will try to fix but no promises ;)

Reply:
> for me the tooltip doesn't work. neither at `threshold`

Hopefully I have fixed tooltips inside dialogs https://github.com/WalletWasabi/WalletWasabi/pull/13800/commits/008057fd99ab1af9214be40b0680f4746930c3bc

Reply:
yep it's fixed

Reply:
We cannot assume our users are completely idiots and we know that people don't read so, forget it.